functions/components/metaTagInjector.ts

```diff
@@ -17,17 +21,19 @@ export class MetaTagInjector {
   element(element) {
     //Open Graph Tags
     this.append(element, 'og:title', this.input.title)
-    this.append(element, 'og:image', this.input.image)
-    this.append(element, 'og:image:width', '1200')
-    this.append(element, 'og:image:height', '630')
+    if (this.input.image) {
+      this.append(element, 'og:image', this.input.image)
+      this.append(element, 'og:image:width', '1200')
+      this.append(element, 'og:image:height', '630')
+      this.append(element, 'og:image:alt', this.input.title)
+      this.append(element, 'twitter:image', this.input.image)
+    }
```

Comment:
Put this with the twitter tags - just do the same `if (this.input.image) {` check down there.